### ISSUE TITLE
"Running emptyrpm inspection...       FAIL" reported incorrectly when using expected_empty to pass the check

### ISSUE THREAD

**Issue by rvykydal** [rpminspect/rpminspect#430]
`FAIL` is reported for `emptyrpm`  in spite of using `expected_empty` making the check actually pass (according to detailed info).
(I cut out downloading messages from the output below)

```
[rvykydal@prnot3 ~/work/rpminspect]$ rpminspect-redhat -p rhel-8-anaconda  -v anaconda-33.16.5.2-1.el8 -o r8.single.dbg.txt
Running license inspection...        pass
Running emptyrpm inspection...       FAIL
Running metadata inspection...       pass
Running manpage inspection...        pass
Running xml inspection...            pass
Running elf inspection...            pass
Running disttag inspection...        pass
Running specname inspection...       pass
Running modularity inspection...     pass
Running javabytecode inspection...   pass
Running ownership inspection...      pass
Running shellsyntax inspection...    pass
Running annocheck inspection...      pass
Running permissions inspection...    pass
Running capabilities inspection...   pass
Running pathmigration inspection...  pass
Running lto inspection...            pass
Running symlinks inspection...       pass
Running files inspection...          pass
Running patches inspection...        pass
LibClamAV Warning: **************************************************
LibClamAV Warning: ***  The virus database is older than 7 days!  ***
LibClamAV Warning: ***   Please update it as soon as possible.    ***
LibClamAV Warning: **************************************************
Running virus inspection...          pass
Running politics inspection...       pass
Running badfuncs inspection...       pass
Running runpath inspection...        pass
[rvykydal@prnot3 ~/work/rpminspect]$ echo $?
0
```
The detailed info from `r8.single.dbg.txt`:
```
emptyrpm:
---------
1) New package anaconda-33.16.5.2-1.el8.aarch64.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



2) New package anaconda-install-env-deps-33.16.5.2-1.el8.aarch64.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



3) New package anaconda-33.16.5.2-1.el8.ppc64le.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



4) New package anaconda-install-env-deps-33.16.5.2-1.el8.ppc64le.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



5) New package anaconda-33.16.5.2-1.el8.i686.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



6) New package anaconda-install-env-deps-33.16.5.2-1.el8.i686.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



7) New package anaconda-33.16.5.2-1.el8.x86_64.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



8) New package anaconda-install-env-deps-33.16.5.2-1.el8.x86_64.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



9) New package anaconda-33.16.5.2-1.el8.s390x.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable



10) New package anaconda-install-env-deps-33.16.5.2-1.el8.s390x.rpm is empty (no payloads); this is expected per the configuration file.
Result: INFO
Waiver Authorization: Not Waivable

```

The profile using `expected_empty`:
```
[rvykydal@prnot3 ~/work/rpminspect]$ sudo cat /usr/share/rpminspect/profiles/redhat/rhel-8-anaconda.yaml
# rpminspect profile for Anaconda

---
inspections:
    # The icon org.fedoraproject.AnacondaInstaller is part of
    # fedora-logos package
    desktop: off

# Fails on changes in .mo files
changedfiles:
    # Ignore updated translations
    ignore:
       - /usr/share/locale/*/LC_MESSAGES/anaconda.mo
emptyrpm:
    # Ignore metapackages
    expected_empty:
        - anaconda
        - anaconda-install-env-deps
        - anaconda-install-img-deps
```

**Comment by dcantrell** [comment]
Great bug report, thank you.  This was caused by a misplaced bool that I was setting in a loop.  One line fix.  Patch coming momentarily.

A comment on using a profile... you don't have to use a profile.  What you have in the profile is entirely ok to add to a local rpminspect.yaml file.  For the purposes of gating in dist-git, you can just add an rpminspect.yaml file to the dist-git repo on the correct branch and put the contents of the profile you created.  The reason I mention this is because if you want to use a profile, then you need to get it added to the rpminspect-data-redhat package and then we also need to modify the pipeline runner for rpminspect to say "if running the job for anaconda, be sure to pass -p rhel-8-anaconda on the command line".  It is probably easier to just add a local rpminspect.yaml to your dist-git repo.

**Comment by rvykydal** [comment]
Thank you. I used a profile for some wrong unrelated reasons, I'll go with the local rpminspect.yaml for sure.
